### PR TITLE
bazel: fix some unit tests

### PR DESCRIPTION
**Pull request by mikedanese**

### CODE REVIEW

File: cmd/kubeadm/test/BUILD [hunk 1]
```diff
@@ -18,6 +18,9 @@ go_test(
     name = "go_default_test",
     srcs = ["token_test.go"],
     library = "go_default_library",
-    tags = ["automanaged"],
+    tags = [
+        "automanaged",
+        "skip",
```
Comment (ixdy) [inline cmd/kubeadm/test/BUILD]: why skip?

File: plugin/pkg/auth/authorizer/rbac/bootstrappolicy/BUILD [hunk 2]
```diff
@@ -26,6 +26,9 @@ go_library(
 go_test(
     name = "go_default_xtest",
     srcs = ["policy_test.go"],
+    data = glob([
+        "testdata/*",
```
Comment (ixdy) [inline plugin/pkg/auth/authorizer/rbac/bootstrappolicy/BUILD]: should we remove the automanaged tag?

Comment (mikedanese) [inline plugin/pkg/auth/authorizer/rbac/bootstrappolicy/BUILD]: automanged doesn't touch data field. This is fine.

File: cmd/kubeadm/test/BUILD (hunk 1 above)
Comment (mikedanese) [inline cmd/kubeadm/test/BUILD]: This is basically an integration test that shells out to kubeadm.

Comment (ixdy) [inline cmd/kubeadm/test/BUILD]: can you leave a comment here noting that?

Comment (mikedanese) [inline cmd/kubeadm/test/BUILD]: Added an integration tag.

**Comment by spxtr** [comment]
/lgtm

@k8s-bot bazel test this

**Pull request merged**